.env

```diff
@@ -0,0 +1,2 @@
+GOOGLE_CLIENT_ID=594164333356-4mh7nfte8udmjtlduk11vviq67ckmlip.apps.googleusercontent.com

```

Comment:
Don’t commit sensitive info. Kill these credentials, make new ones, and make sure this file is gitignored